capuccino/capuccino_py.py

```diff
@@ -0,0 +1,28 @@
+from math import *
+from espresso.espresso_py import espresso
+
+class capuccino(espresso):
```

Comment:
Class names should normally use the CapWords convention.
https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
amended